msteams-platform/apps-in-teams-meetings/create-apps-for-teams-meetings.md

```diff
@@ -322,3 +323,9 @@ The post-meeting and pre-meeting configurations are equivalent.
 
  > [!div class="nextstepaction"]
 > [Meeting token generator app](https://github.com/OfficeDev/microsoft-teams-sample-meetings-token)
```

Comment:
Could you please remove "Meeting app sample" section and add link to code in Code Samples section? Code samples section would look like this:
![image](https://user-images.githubusercontent.com/31851992/111267667-45b71a00-8652-11eb-9af9-837163544892.png)
Source: https://github.com/OfficeDev/microsoft-teams-samples#scenario-specific-samples
Link to use: https://github.com/OfficeDev/Microsoft-Teams-Samples/tree/main/samples/meetings-token-app/csharp